apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/NotificationsTest.cs

```diff
@@ -67,6 +87,9 @@ public void GetServiceAccountEmail()
                 string notificationId = created.Id;
                 Assert.NotNull(notificationId);
 
+                // Wait 30 seconds to let the notification become active reliably.
+                Thread.Sleep(TimeSpan.FromSeconds(30));
```

Comment:
`await Task.Delay()`?

Reply:
Not a lot of different in this case, but sure, why not? :)